sql/modules/Menu.sql

```diff
@@ -102,6 +112,18 @@ It is thus similar to menu_generate() but it only returns the menu items
 associated with nodes directly descendant from the parent.  It is used for
 menues for frameless browsers.$$;
 
+CREATE OR REPLACE FUNCTION menu_item(in_id int) RETURNS SETOF menu_item
+AS $$
+SELECT * FROM menu_generate() where id = $1;
+$$ language sql;
```

Comment:
We do send it at once. As I said before, Same for LedgerSMB::DBObject::Menu::generate_(section|item)
I indended to decide about those after this and **Preferred menu item** PRs